src/scss/components/ec-grid/ec-grid.scss

```diff
@@ -0,0 +1,92 @@
+$gutters: 24px;
+$number-of-columns: 12;
+
+@mixin common {
+  display: flex;
+  justify-content: flex-start;
+  width: calc(100% + #{$gutters});
+  margin-left: calc(#{$gutters} / -2);
+}
+
+.ec-grid {
+  overflow: hidden;
+  width: 100%;
+}
+
+@for $i from 1 through $number-of-columns {
+  .ec-col-#{$i} {
+    margin: calc(#{$gutters} / 2);
+    flex-grow: 0;
+    flex-shrink: 0;
+    flex-basis: calc((100% * ( #{$i} / #{$number-of-columns})) - #{$gutters});
+
+    // For IE11 remove extra 4 px in case we have borders
+    @media all and (-ms-high-contrast: none) {
+      flex-basis: calc((100% * ( #{$i} / #{$number-of-columns})) - #{$gutters} - 4px);
+    }
+  }
+
+  .ec-offset-#{$i} {
+    margin-left: calc(100% * ( #{$i} / $number-of-columns) + #{$gutters});
+  }
+}
+
+.ec-grid__row {
```

Comment:
Why have you change them from `ec-grid--row` to `ec-grid__row`? Does this mean we have to write 3 divs instead of 2?
```
<div class="ec-grid">
  <div class="ec-grid__row">
    <div class="ec-grid__col-1">
```

Reply:
I don't see this a necessarily bad thing.

Reply:
I don't like typing that much, especially I'm used to grids that are using only two layers of divs. The initial version of grid was also doing it that way and I liked it.

Reply:
Yes now we have to use 3 layers. Couldn't make it have offset with 2.